lib/constants.js

```diff
@@ -1,25 +1,62 @@
 const path = require('path');
+const sanitizeFilename = require('sanitize-filename');
+const template = require('lodash-template');
+
 module.exports = {
   experience: {
     commandType: 'experience',
     apiType: 'experienceViews',
     localStatusParams: [ path.join(path.sep, '{components,layouts,pages}', '*.hbs') ],
-    // eslint-disable-next-line no-template-curly-in-string
-    remoteStatusParams: [ path.join('experience', '${viewType}s', '${name}.hbs'), 'body' ]
+    remoteStatusParams: [
+      (resource, { newFilesSet = new Set(), currentFileMap, skipIdCheck = false }) => {
```

Comment:
need to move id check to the top (check all of currentFileMap) and if id match and the view name (not the file name matches) use the current local filename.